rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/activity/OneScanActivity.java

```diff
@@ -20,6 +21,7 @@
 
 public class OneScanActivity extends AppCompatActivity {
 
+    private final long finishDelayed = 4500;
```

Comment:
Could this be a constant and use the uppercase snakecase?

Reply:
It could also be more descriptive like `BARCODE_DETAILS_DISPLAY_TIMEOUT`.

Reply:
fixed.